packages/pwa-buildpack/lib/Utilities/CompatEnvAdapter.js

```diff
@@ -0,0 +1,91 @@
+const ONE_DAY_MS = 24 * 60 * 60 * 1000;
+
+const buildpackVersion = require('../../package.json').version;
+
+class CompatEnvAdapter {
+    static isExpired({ dateChanged, warnForDays }) {
```

Comment:
Quick question JZ. Why did you make this a `static` method instead of a function outside the `class`? Not that it would make any difference, just to pick your brain.

Reply:
It's a good question though. Two reasons:
1. It's a pure function that relies on no instance variables, BUT it's only used by this class
2. I needed to [use it in tests](https://github.com/magento/pwa-studio/pull/2466/files#diff-c71856324fef05d7946db9d0cb69ed5dR48-R52)